test/controllers/transport/WhichRailwayTradeControllerSpec.scala

```diff
@@ -117,6 +122,21 @@ class WhichRailwayTradeControllerSpec extends SpecBase {
 
       application.stop()
     }
+    "save ClaimAmount when 'Painter' is selected" in {
```

Comment:
We are testing vehicle-painters here but we should also test 'vehicle-repairers-wagon-lifters' and 'not-listed' to make sure all cases of the radio button submit the correct value

Reply:
Done